src/app/store/networks/networks.utils.ts

```diff
@@ -64,11 +64,12 @@ export function transformNetworkStateToMultichainStucture(
               },
               bitcoin: {
                 blockchain: 'bitcoin',
-                network: ChainID[chainId].toLowerCase(),
-                url: whenStacksChainId(chainId)({
-                  [ChainID.Mainnet]: BITCOIN_API_BASE_URL_MAINNET,
-                  [ChainID.Testnet]: BITCOIN_API_BASE_URL_TESTNET,
-                }),
+                network: ChainID[chainId] ? ChainID[chainId].toLowerCase() : 'testnet',
+                url:
+                  whenStacksChainId(chainId)({
+                    [ChainID.Mainnet]: BITCOIN_API_BASE_URL_MAINNET,
+                    [ChainID.Testnet]: BITCOIN_API_BASE_URL_TESTNET,
+                  }) || BITCOIN_API_BASE_URL_TESTNET,
```

Comment:
I don't know how we should handle that. The `bitcoin` network doesn't make sense in a subnet context

Reply:
Wouldn't this just be the `chainId.parentNetworkId` instead of `chainId` here? Don't think the `||` is nec?

Reply:
Looks like some more refactoring will have to happen to handle this properly.